packages/nextjs/src/components/pages/contact-step-page/ContactStepPage.module.scss

```diff
@@ -1,45 +1,25 @@
-.contactStepPage {
-    padding-bottom: 1rem;
-    column-gap: 1.75rem;
-    margin: 1.25rem auto 2.5rem;
-    display: grid;
-    grid-template-columns: 1fr 40rem 1fr;
-    grid-template-areas:
-        'pictogram header .'
-        '. content .'
-        '. backLink .';
+.pictogram {
```

Comment:
Kunne man gjenbrukt .pictogram og media query også i MellomstegLayout kanskje?

Reply:
Hmm ja, men da må vi route fra komponenten en annen scss-fil som ikke egentlig tilhører den komponenten, vet ikke om det er så lurt, men vi kunne eventuelt laget en egen komponent for pictogram, en egen for header (det har vi gjort) og en egen for content etc. og så hatt children inni og kjørt stylingen inni de respektive komponentene? Om det gir mening det jeg sier nå etter 10 timer jobb + 2 timer italiensk 😩 snakkes imorgen tror jeg 😂

Reply:
Hmmmm nei da er det kanskje like greit med litt duplikat fortsatt, var ikke veldig gjennomtenkt hvordan vi skulle gjøre det 😅

Reply:
Jeg skal gi det et forsøk, kan hende det kan funke!

Reply:
DONE